### ISSUE TITLE
box of vagrant don't match current version

### ISSUE THREAD

**Issue by oshiro3** [rapid7/metasploitable3#445]
## Issue Description
When I'm going to up win2k8 box as `quick start` following README, I face vagrant error.
It seems `["modifyvm", :id, "--clipboard"]` command executed by win2k8 box is wrong in current vagrant version. 
It require `--clipboard-mode` instead of `--clipboard`.

## Host System
* OS: ubuntu 18..04
* Packer Version: 1.5.1
* Vagrant Version: 2.2.7
* VirtualBox Version: 6.1

## Command Output
```
$ vagrant up win2k8
Bringing machine 'win2k8' up with 'virtualbox' provider...
==> win2k8: Checking if box 'rapid7/metasploitable3-win2k8' version '0.1.0-weekly' is up to date...
==> win2k8: Fixed port collision for 22 => 2222. Now on port 2200.
==> win2k8: Clearing any previously set network interfaces...
==> win2k8: Preparing network interfaces based on configuration...
    win2k8: Adapter 1: nat
    win2k8: Adapter 2: hostonly
==> win2k8: Forwarding ports...
    win2k8: 3389 (guest) => 3389 (host) (adapter 1)
    win2k8: 22 (guest) => 2200 (host) (adapter 1)
    win2k8: 5985 (guest) => 55985 (host) (adapter 1)
    win2k8: 5986 (guest) => 55986 (host) (adapter 1)
==> win2k8: Running 'pre-boot' VM customizations...
A customization command failed:

["modifyvm", :id, "--clipboard", "bidirectional"]

The following error was experienced:

#<Vagrant::Errors::VBoxManageError: There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.

Command: ["modifyvm", "35d946b0-0089-43c0-966a-6eb53107fa21", "--clipboard", "bidirectional"]

Stderr: Oracle VM VirtualBox Command Line Management Interface Version 6.1.2
(C) 2005-2020 Oracle Corporation
All rights reserved.

```

**Comment by greginfosec** [comment]
The issue is with VirtualBox 6.1.  Downgrade to 6.0.16 (december 2019 version) and it'll work as the workaround.  You can leave Vagrant 2.2.7 as is though.  I had to do this.

**Comment by jmartin-r7** [comment]
This should be fixed by #562 which updates the virutalbox requirements to `>= 6.1`
